spec/index.bs

```diff
@@ -925,17 +984,49 @@ has components working on different timelines in parallel:
     on the compute units of the GPU. It includes actual draw, copy,
     and compute jobs that run on the GPU.
 
-<div class=algorithm>
-    This is an example algorithm.
+<div class=example style="background: white">
+    The following show the styling of steps and values associated with each timeline.
+    This styling is non-normative; the specification text always describes the association.
 
-    <div data-timeline=content>
-        Steps executed on the [=content timeline=] look like this.
-    </div>
-    <div data-timeline=device>
-        Steps executed on the [=device timeline=] look like this.
-    </div>
-    <div data-timeline=queue>
-        Steps executed on the [=queue timeline=] look like this.
+    <dl data-timeline=const>
+        : <dfn>Immutable value example definition</dfn>
+        :: Can be used on any timeline.
+    </dl>
+
+    <dl data-timeline=content>
+        : <dfn>Content-timeline example definition</dfn>
+        :: Can only be used on the content timeline.
+    </dl>
+
+    <dl data-timeline=device>
+        : <dfn>Device-timeline example definition</dfn>
+        :: Can only be used on the device timeline.
+    </dl>
+
+    <dl data-timeline=queue>
+        : <dfn>Queue-timeline example definition</dfn>
+        :: Can only be used on the queue timeline.
+    </dl>
```

Comment:
Aren't there pieces of state that we'll need to access on both the device and queue timelines? Or is that all considered implementation details?

I was thinking things like buffer to texture contents, but it looks like we generally refer to those as "contents of |this|" or similar, so maybe not?

Reply:
We will, yes, but we don't have any such state now. As we better-define the queue-timeline state (buffer contents, texture contents, etc.) we can edit this to say "can be used on the device or queue timeline" if we need to, but I don't suspect that's going to be how it works